library/Icingadb/Widget/StateList.php

```diff
@@ -12,6 +12,24 @@ abstract class StateList extends BaseItemList
     use ViewMode;
     use NoSubjectLink;
 
+    protected $hasIconImages = false;
+
+    /**
+     * @return bool
```

Comment:
The method description is missing. Below as well.

Reply:
✅